node/src/accountant/scanners/mod.rs

```diff
@@ -2272,10 +2288,10 @@ mod tests {
             unban_below_gwei: 0,
         };
         let wallet = make_wallet("abc");
-        // It is important to have a payable matching the declining part of the thresholds, also it
-        // will be more believable if the slope is steep because then one second can make the bigger
-        // difference in the intercept value, which is the value this test compare in order to
-        // conclude a pass
+        // It is important to have a payable laying in the declining part of the thresholds, also
+        // it will be the more believable the steeper we have the slope because then a single second
+        // can make a certain difference for the intercept value which is the value this test
+        // compares for carrying out the conclusion
```

Comment:
I believe, these minor improvements will improve the quality of your comment further:

`laying` -> `lying`

`will be the more believable` -> `will be more believable` (without *the*)